august-19.md

```diff
@@ -1,62 +1,151 @@
+* [3.0.3](https://github.com/tikv/tikv/releases/tag/v3.0.3)
+* [3.0.2](https://github.com/tikv/tikv/releases/tag/v3.0.2)
+* [2.1.17](https://github.com/tikv/tikv/releases/tag/v2.1.17)
+* [2.1.16](https://github.com/tikv/tikv/releases/tag/v2.1.16)
 
+Upgrading? Please take note of these things:
 
-## Events
+* In **2.1.17 and 3.0.3** a new `config-check` option allows you to verify a configuration is correct before trying to start a node with it.
+* In **3.0.2** some logs which require no manual intervention (TiKV resolves them automatically) were reduced to `INFO` level.
+* In **2.1.16** `raw_scan` and `raw_batch_scan` now supports reverse.
 
 ## Reading material
 
-TODO: Dongxu posts - pessimistic locking and ...
+Here's some things our contributors have published over the last month:
 
-## Current projects
+* [@sunxiaoguang] published some details of the deployment including TiKV at Zhihu in [Lesson Learned from Queries over 1.3 Trillion Rows of Data Within Milliseconds of Response Time at Zhihu.com](https://pingcap.com/success-stories/lesson-learned-from-queries-over-1.3-trillion-rows-of-data-within-milliseconds-of-response-time-at-zhihu/), some highlights from article:
+    - 1.3 trillion records, growing ~3 billion per day.
+    - 12 million queries processed per second at peak.
+    - Sub-90ms response time.
+* [@aknuds1] published [Practical Networked Applications in Rust, Part 2: Networked Key-Value Store](https://arveknudsen.com/posts/practical-networked-applications-in-rust/module-2/) where he explores the [Rust training course](https://github.com/pingcap/talent-plan/tree/master/rust) created by [@brson] and [@sticnarf].
+* [@siddontang] published [Porting TiDB to ARM64 for Greater Flexibility](https://pingcap.com/blog/porting-tidb-to-arm64-for-greater-flexibility/) where he explores building and benchmarking TiKV and TiDB on ARM64 architectures.
+* [@ethercflow] published a follow up article how [TCP Small Queues become a bottleneck for our test deployments on ARM64 architectures](https://pingcap.com/blog/how-tsq-becomes-a-performance-bottleneck-for-tikv-in-aws-arm-environment/).
+* [@c4pt0r] published [(CN) A Talk about the New Features of TiDB: From Follower Read](https://pingcap.com/blog-cn/follower-read-the-new-features-of-tidb/) where he discusses follower read, a new feature we're planning for TiKV to be leveraged by query layers like TiDB.
+* [@zhangjinpeng1987] published a new [(CN) Source Code Reading: Distributed Transactions](https://pingcap.com/blog-cn/tikv-source-code-reading-12/) discussing the transaction model powering TiKV.
+
+## Roadmap
 
-TODO:
+We've also began nailing down which features we want to include in a future TiKV 4.0 release. Here's a few of our hopes and dreams:
 
-* backup/restore
-* joint consensus
-* pessimistic transactions
+> (This is not an official list and may be subject to change)
 
-## RFCs
+* Publish our RocksDB wrapper as `tirocks` on [crates.io](https://crates.io/crates/tirocks/).
```

Comment:
seem we have not published it yet? only occupy the name

Reply:
Yes! That's why it's on the ROADMAP. :)